README.md

```diff
@@ -48,7 +48,12 @@ development:
  ```
 php vendor/bin/phinx migrate
 ``` 
-  
+* Config bootstrap
+```
+   npm install --global gulp-cli
+   npm install
+   gulp --production
+```
```

Comment:
chỗ này em không cần phải note như này, cái này là lúc em cài đặt thôi em, còn như anh clone về thì em chỉ cần chạy 
```
npm install
gulp 
```
thôi em 
em cho 2 câu lệnh này lên  chỗ
Run this command:
```
   composer install
   npm install
   gulp 
```